spec/api/holdings_api_spec.rb

```diff
@@ -223,28 +220,48 @@ def parse_response(...)
       load_test_data(ht1, ht2, holding)
 
       # as set up above, upenn's holdings (empty enumchron) matches on ht1 both ht2
-      response = parse_response("item_access", v: 1, organization: holding.organization, item_id: ht1.item_id)
+      response = parse_response("item_access", organization: holding.organization, item_id: ht1.item_id)
       expect(response["copy_count"]).to eq 1
-      response = parse_response("item_access", v: 1, organization: holding.organization, item_id: ht2.item_id)
+      response = parse_response("item_access", organization: holding.organization, item_id: ht2.item_id)
       expect(response["copy_count"]).to eq 1
     end
+
```

Comment:
These are the added tests for `brlm_count`.

Reply:
We should probably have a test for an MPM...

Reply:
Yep. Added.